### ISSUE TITLE
temporary HP bar not showing correctly when max HP is lower than temporary HP

### ISSUE THREAD

**Issue by dmraydm** [kamejosh/owlbear-hp-tracker#42]
Same as what the title says, temporary HP bar not showing correctly when max HP is lower than temporary HP. The HP bar turns completely blue and keeps extending to the right outside of the box width it normally stays within.
![Screenshot_20240210_195057](https://github.com/kamejosh/owlbear-hp-tracker/assets/44980485/38d5b55d-138a-46bf-ab73-6867037b7fcb)

**Comment by kamejosh** [comment]
Yes, this behaviour is by design. Because all healthbars get their length as a percentage of the max HP. So the temp HP are 3 times the max HP which means they get a length of 300% of the hp bar. 

The alternative would be to have it at max 100% but then people would complain that it shows too little because it should actually be longer.

Do you have a better idea to solve this problem?

**Comment by dmraydm** [comment]
I appreciate the response, I understand what you mean and that makes sense. I think you're right, the way this is currently programmed may be the best solution with this situation because it may be unlikely that many people will run in to this issue at all.

I've dabbled in programming in the past but I am no expert- the first idea that comes to mind for a solution here could be to keep the bar the same width it normally is, and invert the colors. So in my example above, the HP bar would be 1/4 red and 3/4 blue (1/4 of the shown current total HP of 12 is equal to max HP of 3 and 3/4 of current total HP is equal to current temp HP of 9)

To put it another way, instead of extending the blue bar further to the right, the blue bar would instead push the red bar smaller as the temp HP goes up, keeping both bars still confined in that width while still visually representing what % of each bar makes up the total combined current HP of both max and temp.

I apologize if this is explained poorly or if I am missing anything obvious, I appreciate your work and this extension is awesome, so I completely understand if this particular issue isn't a priority. I enjoy thinking it through at least!
